components/DeckPreview.tsx

```diff
@@ -0,0 +1,30 @@
+import Link from 'next/link'
+
+import { rarityColorKey } from '../lib/utils'
+import { IDeck } from '../lib/types'
+import * as CSS from 'csstype'
```

Comment:
How strongly do you feel about introducing `csstype`? I'm down but wanted to double check that you prefer it over tailwind :D

Reply:
it looks like we already had it included with our `yarn.lock` file, so i figured it was a good call. i love tailwind, and am a hold out from typescript

https://github.com/Adventure-Cards/deck-viewer/pull/4/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deR802